src/content/docs/browser/new-relic-browser/configuration/experimental-features.mdx

```diff
@@ -36,3 +38,39 @@ The following experimental features are available in New Relic Browser:
   } }:
   ```
 4. Deploy your app.
+
+### NPM Implementations
+
+#### Browser Performance - Marks, Measures & Resources
+1. Ensure you are using a version of the New Relic Browser agent compatible with the experimental feature.
+2. Find the New Relic browser agent constructor in your application's implementation.
+3. In the `init` configuration object, add the `performance` feature configuration. Here's an example that enables both marks and measures detection:
+```js
+import { BrowserAgent } from '@newrelic/browser-agent/loaders/browser-agent'
+
+// Populate using values in copy-paste JavaScript snippet.
+const options = {
+  init: {
+    // ... other configurations
+    performance: {
+      capture_marks: true, // enable to capture browser performance marks (default false)
+      capture_measures: true // enable to capture browser performance measures (default false)
+      resources: {
+        enabled: true, // enable to capture browser peformance resource timings (default false)
+        asset_types: [], // Asset types to collect -- an empty array will collect all types (default []). See https://developer.mozilla.org/en-US/docs/Web/API/PerformanceResourceTiming/initiatorType for the list of types.
+        first_party_domains: [], // when included, will decorate any resource as a first party resource if matching (default [])
+        ignore_newrelic: true // ignore capturing internal agent scripts and harvest calls (default true)
+      }
+    }
+   },
+  info: { ... },
+  loader_config: { ... } 
+}
+
+// The agent loader code executes immediately on instantiation.
+new BrowserAgent(options)
+```
+See the [NPM package documentation](https://www.npmjs.com/package/@newrelic/browser-agent) for more information on how to configure the agent via NPM.
+
+## Opting-in for APM injected applications
+APM-served web applications can opt-in to experimental features by reaching out to your support representative, by filing a help ticket or by emailing `browser-agent@newrelic.com` with a subject line starting with `[Experimental Features]: `.
```

Comment:
For APM-served web applications, you can opt in to experimental features. Contact your support representative, file a help ticket, or email browser-agent@newrelic.com with a subject line starting with [Experimental Features]: .